wlauto/workloads/excel/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -71,7 +87,17 @@ private void newFile() throws Exception {
         newButton.click();
     }
 
-    private void createInTestFolder() throws Exception {
+    private void openWorkbook(final String filename) throws Exception {
+        UiObject openButton = getUiObjectByText("Open", "android.widget.Button");
+        openButton.click();
+
+        navigateToTestFolder();
+
+        UiObject fileEntry = getUiObjectByText(filename, "android.widget.TextView");
+        fileEntry.click();
```

Comment:
It would be good to time the taken to click and open the file.  This will show any file system performance deltas between devices.